modules/core/src/main/java/org/apache/ignite/internal/processors/rest/GridRestCommand.java

```diff
@@ -176,6 +176,18 @@
     /** */
     CLUSTER_CURRENT_STATE("currentstate"),
 
+    /** */
+    BASELINE_CURRENT_STATE("baseline"),
+
+    /** */
+    BASELINE_SET("baselineset"),
```

Comment:
According to what I see in this class we should probably use names like "baselinestate", "setbaseline", "addbaseline" and "removebaseline".